adops/js-dynamic-creative.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page_v2
+title: Prebid.js dynamic creatives
+head_title: Prebid.js dynamic creatives 
+sidebarType: 3
+---
+
+# Prebid.js dynamic creatives
+{:.no_toc}
+
+- TOC
+{: toc}
+
+## Overview
+
+For line items that target browsers only, you may use Prebid.js dynamic creatives as an alternative to [Prebid Universal Creative](/overview/prebid-universal-creative.html). The former provides slightly better performance and ease of use, but relies on Prebid.js, meaning that it does not support non-JS use cases (such as mobile apps or AMP).
+
```

Comment:
Does the dynamic creative support `banner` and `video` as media type or only `banner`?

Reply:
it supports everything PUC does as long as it's talking to JS. so you could say it supports video but really what it does is ask Prebid to run a renderer (like PUC).

Reply:
Maybe that's worth putting in the docs as well 😊